R/utils.R

```diff
@@ -93,14 +93,10 @@
   # Validate polarization and colocalization
   if (length(polarization) > 0) {
     # Check column names
-    stopifnot(
-      "'polarization' names are invalid" =
-        all(sort(names(polarization)) ==
-          sort(c(
-            "morans_i", "morans_p_value", "morans_p_adjusted",
-            "morans_z", "marker", "component"
-          )))
-    )
+    name_check <- all(c("marker", "component") %in% names(polarization))
+    if (!name_check) {
+      abort("Columns 'marker', and 'component' are required in the 'polarization' score table")
```

Comment:
Nice, this particular check caused some issues before when loading files from different pixelator runs. Should work now for most cases.